test/metric_learn_test.py

```diff
@@ -72,6 +74,37 @@ def test_iris(self):
       self.assertLess(csep, 0.25)
 
 
+def test_convergence_simple_example(capsys):
+  # LMNN should converge on this simple example, which it did not with
+  # this issue: https://github.com/metric-learn/metric-learn/issues/88
+  X, y = make_classification(random_state=0)
+  lmnn = python_LMNN(verbose=True)
+  lmnn.fit(X, y)
+  out, _ = capsys.readouterr()
+  assert "LMNN converged with objective" in out
+
+
+def test_no_twice_same_objective(capsys):
```

Comment:
Should these be methods of `TestLMNN`?

Reply:
I read here (https://github.com/pytest-dev/pytest/issues/2504#issuecomment-308828149) that pytest fixtures cannot be integrated with unittest classes, so I extracted the tests from the class hierarchy. But I agree that it is not ideal. They propose a workaround in the link, so maybe it would be better ? (adding these lines to `TestLMNN`, include the test in `TestLMNN`, and replace capsys by self.capsys in the test)
```python
@pytest.fixture(autouse=True)
def capsys(self, capsys):
  self.capsys = capsys
```